tests/kernels/test_cache.py

```diff
@@ -362,7 +362,7 @@ def test_swap_blocks(
     block_mapping = list(zip(src_blocks, dst_blocks))
     block_mapping_tensor = torch.tensor(block_mapping,
                                         dtype=torch.int64,
-                                        device="cpu").view(-1, 2)
+                                        device=device).view(-1, 2)
```

Comment:
Is this because that this tensor need to be accessed by the new CUDA memcpy kernel?

Reply:
Yes. The new `paged_copy` kernel need to access the block mapping from GPU.